src/libs/actions/Wallet.js

```diff
@@ -126,17 +232,48 @@ function activateWallet(currentStep, parameters) {
 
                     if (response.title === CONST.WALLET.ERROR.FULL_SSN_NOT_FOUND) {
                         setAdditionalDetailsShouldAskForFullSSN(true);
+                        setAdditionalDetailsErrorMessage(Localize.translateLocal('additionalDetailsStep.needSSNFull9'));
+                        return;
                     }
 
-                    const errorTitles = [
-                        CONST.WALLET.ERROR.FULL_SSN_NOT_FOUND,
-                        CONST.WALLET.ERROR.IDENTITY_NOT_FOUND,
-                        CONST.WALLET.ERROR.INVALID_SSN,
-                        CONST.WALLET.ERROR.UNEXPECTED,
-                        CONST.WALLET.ERROR.UNABLE_TO_VERIFY,
-                    ];
+                    let qualifiers = _.get(response, ['data', 'requestorIdentityID', 'apiResult', 'qualifiers', 'qualifier'], []);
```

Comment:
Oh yeah small thing but we have [standardized](https://github.com/Expensify/App/blob/main/STYLE.md#accessing-object-properties-and-default-values) on using `lodashGet()`. A `get()` was added to underscore in a recent version, but doesn't have the path based syntax like (`lodashGet(response, 'data. requestorIdentityID.apiResult.etc')`) so we avoid using it to keep things consistent.